### PR TITLE
Update rubygem-foreman_ansible to 7.0.0

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Should get into 3.1

DEB: #7386

### CODE REVIEW

File: packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -46,6 +44,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@testing-library/user-event) >= 13.2.1
+BuildRequires: %{?scl_prefix}npm(@testing-library/user-event) < 14.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:48]:
```
DEBUG util.py:600:  No matching package to install: 'npm(@testing-library/user-event) >= 13.2.1'
DEBUG util.py:600:  No matching package to install: 'npm(@testing-library/user-event) < 14.0.0'
```

is this something we actually need? or is this for testing only?

aka: should we add `@testing-library` to excluded prefixes in https://github.com/theforeman/foreman/blob/develop/package-exclude.json?

Comment (evgeni) [reply packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:48]: looking at https://testing-library.com/, excluding all these packages seems sane?

Comment (ezr-ondrej) [reply packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:48]: yeah, I think it make sense :+1: Will do

**Comment by evgeni** [comment]
This is now failing on the graphql loader issue.

You can workaround it like we did in https://github.com/theforeman/foreman-packaging/pull/7291 until we properly fix https://projects.theforeman.org/issues/34086

**Comment by evgeni** [comment]
```
ModuleNotFoundError: Module not found: Error: Can't resolve '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon' in '/builddir/build/BUILDROOT/rubygem-foreman_ansible-7.0.0-1.fm3_2.el8.noarch/usr/share/gems/gems/foreman_ansible-7.0.0/webpack'
```

Hmmm.... I recall we had this issue in the past. Using `dist` is the wrong way here.

Need to dig foreman core git history for the fix.

**Comment by evgeni** [comment]
https://github.com/theforeman/foreman_ansible/pull/492

**Comment by ezr-ondrej** [comment]
Leapp fails, it is probably not an issue of this PR, right?

**Review by evgeni — APPROVED** [review]

**Comment by ezr-ondrej** [comment]
CP 3.1: #7397

**Pull request merged**